devops/iam-did-auth-proxy-staging/values.yaml

```diff
@@ -0,0 +1,284 @@
+
+  sealedSecret:
+    enabled: true
+    annotations: {}
+    encryptedData:
+      IPFS_PROJECTSECRET: AgAIpovKHlyFYxOT6Nk4d+T6m9zKF95ZdUgQPx/fsWZw1aAQJDXzDL9CZBI++zZ4pXnra4L4EftL8hyJwKPqSSVp9IjB+IcOb2wC3EaE4IEiMl2HGbzpqKtrmMoik4NkPbC1N7pwowuO43tttE2eUZblz85+3g+ccCTz/GouzGY/ryBKpVADGEafrrbtv6X41NsGYpe+Pzu6fApjIP9gJzkdwW8nIo8l5sUqfSZHkIKlGvJuuIp29l5IKeQyQemvHk0pD4/MAOmzYoe8my6ZO/6c9l4SBnbtG7BqrIqhKlRN4hw3RINVC6bt6dkSuznlZRos24pAKOufPoRcZvdIUti2a5KheXLCVxO0l99sBOlW674mT2Qim+jdORNxAJbt3pUhy/xHwWxxBWn++J7WAgvOYpihSd/bKsKm2JcMlvHKvjFY2hGzjPlrAP38BiJKgC4SMIA3DAouylt/6y8U9Fwd2d4GnPCsKPewBvqSCdxMgT331tfV/BeTaHV70WcE02k8qX1xOrFHkmiaSQo1kimh9KCe/p9gB1EuGeN8Z3bbUbcAd8n4P4Ry0pIUaaF/VNqi4eM3+Kpgx8aS5fa8o9iKyYVKMPNTzIKEPYp63u+TQmOUX1Kj8M/67ZqItB0XiHyocVFcTdDfITrGPjilxwyxZJrrNWWIiqUFCfSJhZelBnQ+qrGrHOUNdmq29uHYKf2w5MoXwGXmeSLNbiur9QghKtl3QJPDQFTTn2Ulv/aThQ==
+      
+  ########
+  # Redis
+  redis:
+    architecture: standalone
+    auth:
+      password: redis
+      
+  ########
+  # nginx
+  nginx:
+    fullnameOverride: "did-auth-proxy-helm-server-block"
+
+    serverBlock: |-
+      map $http_origin $allow_origin {
+        default "";
+        "~^(http://localhost:4200|http://localhost:4201|https://gp4btc-miner-stg.energyweb.org|https://gp4btc-explorer-stg.energyweb.org)$" "$http_origin";
+      }
+        
+      map $request_method $cors_method {
+        default "allowed";
+        "OPTIONS" "preflight";
+      }
+        
+      map $cors_method $cors_max_age {
+        default "";
+        "preflight" 1;
+      }
+        
+      map $cors_method $cors_allow_methods {
+        default "";
+        "preflight" "GET, POST, OPTIONS";
+      }
+        
+      map $cors_method $cors_allow_headers {
+        default "";
+        "preflight" "Authorization,Content-Type,Accept,Origin,User-Agent,DNT,Cache-Control,X-Mx-ReqToken,Keep-Alive,X-Requested-With,If-Modified-Since";
+      }
+        
+      map $cors_method $cors_content_length {
+        default $initial_content_length;
+        "preflight" 0;
+      }
+        
+      map $cors_method $cors_content_type {
+        default $initial_content_type;
+        "preflight" "text/plain charset=UTF-8";
+      }
+      
+      server {
+        listen 0.0.0.0:8080;
+        server_name gp4btc-backend-proxy-stg.energyweb.org;
+
+        location /auth {
+            proxy_pass http://did-auth-proxy-helm.iam-did-auth-proxy.svc.cluster.local;
+        }
+
+        location /api-auth {
+            proxy_pass http://did-auth-proxy-helm.iam-did-auth-proxy.svc.cluster.local;
+        }
+
+        location /auth/healthcheck {
+            proxy_pass http://did-auth-proxy-helm.iam-did-auth-proxy.svc.cluster.local/healthcheck;
+        }
+
+        location /api {
+            proxy_pass http://gp4btc-backend-microservice.gp4btc.svc.cluster.local;
+        }
+
+        location / {
+            auth_request /token_introspection;
+            proxy_pass http://gp4btc-backend-microservice.gp4btc.svc.cluster.local;
```

Comment:
Just doublechecking: is it expected that these hostnames are the same for all environments?